src/Culling/babylon.boundingBox.ts

```diff
@@ -1,4 +1,7 @@
-﻿module BABYLON {
+module BABYLON {
+
+    const _identityMatrix = Matrix.Identity();

```

Comment:
is const comparable to static readonly?

Reply:
nope, const just ensures at "compile time" that a variable is not changed

Reply:
ok so I would encourage using static readonly to save memory

Reply:
yup, I have no preference.
In both cases the Matrix is created only once. Having a closure on an unreachable and locally declared variable has the advantage of not "polluting" the class prototype.

I did it this way because that's what we were doing in BoundingSphere